src/clj_foundation/errors.clj

```diff
@@ -109,3 +106,65 @@ attempts, the last caught exception is re-thrown."
   The remaining parameters (if any) are that function's arguments."
   [tries pause-millis]
   (fn [f & args] (apply retry tries pause-millis f args)))
+
+
+(s/defn replace-if :- s/Any
```

Comment:
OK; refactoring point noted.  

I don't agree because errors.clj was always a library module for error handling.  Whether it lived inside icarrp.util or clj_foundation was (and is) irrelevent.

I suggest that if you feel strongly about this that we take that up as a separate refactoring ticket.